app/build.gradle

```diff
@@ -4,13 +4,26 @@ buildscript {
     ext.protobufVersion = '3.21.7'
     ext.jerseyVersion = '3.1.0'
     ext.junitVersion = '5.9.0'
+    ext.postgresVersion = '42.5.1'
+    ext.jooqVersion = '3.17.4'
+    ext.guiceVersion = '5.1.0'
+    dependencies {
+        classpath  "org.postgresql:postgresql:$postgresVersion"
+    }
+
+    configurations['classpath'].resolutionStrategy.eachDependency {
```

Comment:
what does this do?

Reply:
removed in next commit. 
it was required to force dependency resolution to particular version.